.gitignore

```diff
@@ -0,0 +1 @@
+.idea
```

Comment:
This should be moved to your global `.gitignore`. If we support 1 platform, we must support all, and there are very very many.